Cargo.lock

```diff
@@ -4264,6 +4308,59 @@ dependencies = [
  "thiserror",
 ]
 
+[[package]]
+name = "prost"
+version = "0.9.0"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "444879275cb4fd84958b1a1d5420d15e6fcf7c235fe47f053c9c2a80aceb6001"
+dependencies = [
+ "bytes",
+ "prost-derive",
+]
+
+[[package]]
+name = "prost-build"
```

Comment:
Huh, it makes no sense I think that we pull code generation here... I would expect the rust code to be pre-generated...

Will ask tonic folks about this.